HotelManagement.Application.Implements/HotelService.cs

```diff
@@ -92,6 +111,15 @@ private static HotelDetailsDto MapToDetailsDto(Hotel hotel)
             };
         }
 
+        private static HotelImagesDto MapToImagesDto(Hotel hotel)
+        {
+            return new HotelImagesDto
+            {
+                Id = hotel.Id,
+                Image = hotel.Images
```

Comment:
Should be seperated into a HotelImageDto with following structure : 
[
{
    "name" : "image-name",
    "url" : "image-url"
}
]